sstables/sstables.cc

```diff
@@ -1765,7 +1765,7 @@ future<> sstable::seal_sstable(bool backup)
             _marked_for_deletion = mark_for_deletion::none;
         }
         if (backup) {
-            return _storage.snapshot(*this, get_dir() + "/backups/");
+            return _storage.snapshot(*this, "backups", filesystem_storage::absolute_path::no);
```

Comment:
out of scope, but we should refrain from using string literals for sub-directory names, but rather define it as a constexpr in sstables.hh alongside `snapshots_dir`